kotlin/src/main/kotlin/io/logto/sdk/core/Core.kt

```diff
@@ -50,4 +63,70 @@ object Core {
             }
         }.build().toString()
     }
+
+    fun fetchOidConfig(endpoint: String, completion: HttpCompletion<OidcConfigResponse>) =
+        httpGet(endpoint, completion)
+
+    @Suppress("LongParameterList")
+    fun fetchTokenByAuthorizationCode(
+        tokenEndpoint: String,
+        clientId: String,
+        redirectUri: String,
+        codeVerifier: String,
+        code: String,
+        resource: String?,
+        completion: HttpCompletion<CodeTokenResponse>
+    ) {
+        val body = JsonObject().apply {
+            addProperty(QueryKey.CLIENT_ID, clientId)
+            addProperty(QueryKey.REDIRECT_URI, redirectUri)
+            addProperty(QueryKey.CODE_VERIFIER, codeVerifier)
+            addProperty(QueryKey.CODE, code)
+            addProperty(QueryKey.GRANT_TYPE, GrantType.AUTHORIZATION_CODE)
+            resource?.let { addProperty(QueryKey.RESOURCE, it) }
+        }.toString().toRequestBody(MediaType.X_WWW_FORM_URLENCODED.toMediaType())
+        httpPost(tokenEndpoint, body, completion)
+    }
+
+    @Suppress("LongParameterList")
+    fun fetchTokenByRefreshToken(
+        tokenEndpoint: String,
+        clientId: String,
+        refreshToken: String,
+        resource: String?,
+        scope: List<String>?,
```

Comment:
Would you let me know why scope is an string array here? I thought it would be string.

Reply:
OK, after discussion, we decided to update the document to use string array for `scope` param, so that all platform SDKs would be aligned. Cheers